pkg/geodata/ckmeans.go

```diff
@@ -27,15 +27,15 @@ AND geo.valid
 
 -- the category we are interested in
 AND nomis_category.long_nomis_code = $2
-AND nomis_category.year = 2011
+AND nomis_category.year = $3
```

Comment:
as @stmuk points out,  would it make more sense to have `nomis_category.year = data_ver.year` and `data_ver.year = $3` here?